doc.go

```diff
@@ -63,6 +63,25 @@ Or it can be loaded from a file using -f or --file:
 
 	$ yab -p localhost:9787 -t kv.thrift kv KeyValue::Get --file req.yaml
 
+Request options can also be specified in a YAML file, e.g., get.yab:
+	#!/usr/bin/env yab -y
```

Comment:
Also, I wouldn't introduce the shebang first. Just the `yab -y foo.yaml` format and then say, `-y` reads input from stdin if a filename wasn't provided so it can be used as a `shebang` for executable YAML files like so. Otherwise the assumption will be that the templates must have the shebang.